src/Sarif/Core/FileLocation.cs

```diff
@@ -20,7 +20,7 @@ public partial class FileLocation
         /// <returns></returns>
         public bool TryReconstructAbsoluteUri(IDictionary<string, Uri> originalUriBaseIds, out Uri resolvedUri)
         {
-            resolvedUri = null;
+            resolvedUri = this.Uri.IsAbsoluteUri ? this.Uri : null;
```

Comment:
>IsAbsoluteUri [](start = 35, length = 13)

we missed a unit test here. please drop one in quickly? just a single test for this single change. will create a placeholder for us to add to.